pkg/templates/crds/cluster-lifecycle/cluster.open-cluster-management.io_clustercurators.yaml

```diff
@@ -1,9 +1,10 @@
+
```

Comment:
```suggestion
```
lets give rid of this extra space just in case.

Reply:
Fixed. Thanks!